art.taunoerik.tauno-serial-plotter.desktop

```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Version=1.0
+Encoding=UTF-8
+Type=Application
+Name=Tauno-serial-plotter
+Comment=Simple serial plotter
+TryExec=tauno-serial-plotter
+Exec=tauno-serial-plotter
```

Comment:
You probably want to add "field code" to pass a file name to be open, if you application actually support it from the command line. See:
https://specifications.freedesktop.org/desktop-entry-spec/desktop-entry-spec-latest.html#exec-variables